config/locales/en.yml

```diff
@@ -85,7 +86,7 @@ en:
     confirmation:
       missing: missing
       not_provided: Not provided
-      heading: "Confirm trainee’s %{section_title}"
+      heading: "Confirm %{section_title}"
```

Comment:
I'm not sure about changing this - can we keep it as it is for now? I think the prototype might be out of date.